npu.user.js

```diff
@@ -606,8 +606,12 @@
         }
         var keepAlive = function() {
           window.setTimeout(function() {
+              var pages = ["main.aspx", "main.aspx?ismenuclick=true&ctrl=inbox", "main.aspx?ctrl=0203&ismenuclick=true", "main.aspx?ismenuclick=true&ctrl=outbox"];
+              var index = Math.floor(Math.random() * Math.floor(pages.length));
+              var targetUrl = pages[index-1];
```

Comment:
`index` is in [0,`pages.length`), which is exactly the range `pages` has useful information in. You don’t need to subtract one from it.